frontend/public/module/k8s/cluster-settings.ts

```diff
@@ -39,6 +39,29 @@ export const getSortedUpdates = (cv: ClusterVersionKind): ClusterUpdate[] => {
   }
 };
 
+export const getNewerMinorVersionUpdate = (currentVersion, availableUpdates) => {
+  const currentVersionParsed = semver.parse(currentVersion);
+  return availableUpdates?.find(
+    // find the next minor version update, which there should never be more than one
+    (update) => {
+      const updateParsed = semver.parse(update.version);
+      return (
+        `${updateParsed.major}.${updateParsed.minor}` >
+        `${currentVersionParsed.major}.${currentVersionParsed.minor}`
+      );
+    },
+  );
+};
+
+export const getMinorVersionIsNewer = (currentVersion, otherVersion) => {
+  const currentVersionParsed = semver.parse(currentVersion);
+  const otherVersionParsed = semver.parse(otherVersion);
+  return (
+    `${otherVersionParsed.major}.${otherVersionParsed.minor}` >
+    `${currentVersionParsed.major}.${currentVersionParsed.minor}`
+  );
```

Comment:
It's not safe to compare these as strings

```
>> "4.10" > "4.9"
<- false
```

The semver library has `lt` and `gt` functions for comparing versions.